modules/ztp-configuring-hwevents-using-pgt.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-creating-hwevents_{context}"]
+= Configuring bare-metal events monitoring using PolicyGenTemplate CRs
+
+You can configure bare-metal hardware events for vRAN clusters that are deployed using the GitOps Zero Touch Provisioning (ZTP) pipeline.
+
+.Prerequisites
```

Comment:
Bare Metal Event Relay feature requires AMQ Interconnect to be installed. Similar to what's in modules/ztp-configuring-ptp-fast-events.adoc. We can either add this as a prerequisites and refer to ptp doc or copy the following procedures to here:

Add the following YAML into .spec.sourceFiles in the common-ranGen.yaml file to configure the AMQP Operator:
#AMQ interconnect operator for fast events
- fileName: AmqSubscriptionNS.yaml
  policyName: "subscriptions-policy"
- fileName: AmqSubscriptionOperGroup.yaml
  policyName: "subscriptions-policy"
- fileName: AmqSubscription.yaml
  policyName: "subscriptions-policy"
  
Apply the following PolicyGenTemplate changes to your specific site YAML files, for example, example-sno-site.yaml:

In .sourceFiles, add the Interconnect CR file that configures the AMQ router to the config-policy:

- fileName: AmqInstance.yaml
  policyName: "config-policy"

Reply:
@jzding I would prefer not to copy anything that is already documented. I added both Operators as prereqs and links to the installing procedures. Let me know what you think!